astro/src/content/blog/2023-hacktoberfest.mdx

```diff
@@ -0,0 +1,52 @@
+---
+publish_date: 2023-10-04
+title: Hacktoberfest 2023 with FusionAuth 
+description: FusionAuth joins in the Hacktoberfest event in 2023
+authors: Alex Patterson
+image: /img/blogs/2023-hacktoberfest/hacktoberfest-2023.png
+categories: News
+tags: github, hackathon
+excerpt_separator: "{/* more */}"
+---
+import RemoteCode from '../../components/RemoteCode.astro';
+
+We're excited to announce FusionAuth's participation in this year's Hacktoberfest! Hacktoberfest is a global event that takes place every October, during which developers are encouraged to contribute to open-source projects like FusionAuth. The goal is to foster a vibrant open-source community, celebrate shared knowledge, and make the world of coding more accessible to all. You can find other participating repositories by searching the <a href="https://github.com/topics/hacktoberfest" target="_blank" rel="noopener noreferrer">hacktoberfest topic</a> on github.
+
+{/* more */}
+
+## Getting Started
+
+The first step is to <a href="https://hacktoberfest.com/register" target="_blank" rel="noopener noreferrer"> register for Hacktoberfest</a>. Once you've registered, any PRs submitted to FusionAuth will count towards your participation in the event!
+
+## Contributing
+
+If you haven't participated before make sure to read the <a href="https://hacktoberfest.com/participation/ " target="_blank" rel="noopener noreferrer"> Hacktoberfest Participation Guide</a>.
+
+To see all of FusionAuth's issues in one place see the [full issue search](https://github.com/search?q=user%3AFusionAuth+label%3Ahacktoberfest+state%3Aopen&type=Issues&ref=advsearch&l=&l=).
+
+This year for the Hacktoberfest we choose to focus on all of our JavaScript repositories. I have added the tag `hacktoberfest` to each one of the following repositories and even added an issue for simply updating the packages to their latest version. 
+
+
+* [Angular ](https://github.com/fusionauth/fusionauth-quickstart-javascript-angular-web)- The Authorization Code grant using the Angular framework
+* [Consents ](https://github.com/FusionAuth/fusionauth-example-express-consents)- Example using advanced1 registration forms and consents
```

Comment:
```suggestion
* [Consents ](https://github.com/FusionAuth/fusionauth-example-express-consents)- Example using advanced registration forms and consents
```

This requires a license, not sure if you should note that. We could also grant everyone a starter license and then revoke it after the month.